sdk/src/wallet/core/mod.rs

```diff
@@ -360,14 +345,22 @@ where
     crate::wallet::Error: From<S::Error>,
     crate::client::Error: From<S::Error>,
 {
+    // TODO: Is this method needed? It calls `get_default_sync_options` which the builder does not.
```

Comment:
Looks like this got messed up in the single account merge since it was an account
`get_default_sync_options()` should be loaded in the wallet builder now

Reply:
Opened an issue for it https://github.com/iotaledger/iota-sdk/issues/2022

Reply:
and there it goes ....